docs/ossec_alerts.rst

```diff
@@ -90,39 +90,44 @@ install\_files/ansible-base and then specify the filename in the
 If you don't have your GPG key ready, you can run GnuPG on the command
 line in order to find, import and export your public key. It's best to
 copy the key from a trusted and verified source, but you can also
-request it from keyservers using the known longid. Looking it up by
-email address could cause you to obtain a wrong, malicious or expired
-key. Just replace the key ID in the following example with yours:
+request it from keyservers using the known fingerprint. Looking it up by
+email address or a shorter key ID format could cause you to obtain a
+wrong, malicious or expired key. Just replace the word ``<fingerprint>``
+in the following example with your own:
 
-Download your key and import it into the local keyring:
+Download your key and import it into the local keyring: ::
 
-::
+    gpg --keyserver <fingerprint> hkp://qdigse2yzvuglcix.onion --recv-key C5986B4F1257FFA86632CBA746181433FBB75451
 
-    gpg --recv-keys 0xFD720AD9EBA34B1C
+As in the setup for the *Admin Workstation*, we encourage you to first
+write out the fingerprint in groups of four, double-check you have
+entered it correctly, remove the spaces, and then press "enter" to
+actually fetch the key (and implicitly verify the fingerprint of the key
+received matches the one requested).
 
-Export the key to a file:
+Next we export the key to a local file. It's okay to type the email
+address your key is associated with in the place of ``<fingerprint>`` in
+the command below. Since it is a local operation we don't have to worry
+about the network-based attacks we do during import from a keyserver. ::
 
-::
+    gpg --export -a <fingerprint> > ossec.pub
```

Comment:
If you're going to allow the use of an email address here, I think you should replace `<fingerprint>` with `<keyid>`. Fingerprints are something specific, and it doesn't _really_ make sense to say that they are interchangeable with email addresses.

Key IDs, on the other hand, are defined as anything that can identify a specific key, which includes names, email addresses, truncated fingerprints ("short" and "long" key ids) and of course fingerprints.